atlas/knowledge-environment/docker-compose.prod.yml

```diff
@@ -228,7 +228,22 @@ services:
         aliases:
           - arranger-server
 
-  
+
```

Comment:
It should not live in the prod compose. We don't need/want it running all of the time.